test/runtests.jl

```diff
@@ -1,5 +1,9 @@
 using Clang
 using Test
+using Pkg
 
 include("generators.jl")
+
+include("test_mpi.jl")
+Pkg.build("CMake")
```

Comment:
Local tests were giving `CMake` errors.

Reply:
Why use CMake here?

Reply:
No, it's for the following test `test_bitfield.jl` which uses `CMake`.

Reply:
IIRC, building CMake.jl should be done when installing a package, not when testing it.

Reply:
Yeah well, this errors out locally. Will try to remove for remote `ci`.

Reply:
I've removed it, CI looks ok.